README.md

```diff
@@ -1 +1,50 @@
-# angular-what-is-angular-readme
+# What is Angular and why does it exist?
+
+## Overview
+
+In this lesson we're going to be looking into AngularJS (commonly referred to as just "Angular"), an open-source framework that was released in 2009. It is currently being mainly maintained by Google as well as a massive community of developers.
+
+It aims to solve the problems faced when creating single page applications, implementing a MVVM pattern to separate presentation and business logic.
+
+## Objectives
+
+- Describe what Angular's role in the web platform is
+- Describe the issues Angular looks to solve
+
+## Frontend Frameworks
+
+Ever wondered how Facebook can have a different timeline for millions of users? How Twitter can display tweets for 300 million people? That's where frontend frameworks come in. 
+
+Frontend frameworks have grown massively in the last 3 years. They help developers structure their applications and create interactive, super quick applications.
+
+### Life before frameworks
+
+Before we had the pleasure of using frontend frameworks, code would be written very procedurally. JavaScript code would become unwieldy as there was no easy way to structure code for each page that the client would see.
+  
+You'd be limited to just simple JavaScript interactions, such as submitting a form via AJAX. Anything more wouldn't have been scalable, resulting in a hard to debug/manage codebase. 
```

Comment:
In this context, it's unclear what 'scalable' means. Try to give an example of why this isn't scalable. For example, without actually getting into the nitty-gritty or very technical, what would it look like to try to build Gmail without a front end framework? Duplicated code? Difficult to read and maintain functions? Discuss specific apps/features rather than saying "anything more"

Discuss some of the other problems of not using front-end frameworks - duplicated code? Spending more time with setup and less time actually writing your application? Be specific with your examples but not too technical.